docs/Configuration.md

```diff
@@ -27,6 +28,38 @@ MySQL             | `MySQL`
 PostgreSQL        | `PostgreSQL`
 SQLite            | `SQLite`
 
+### user
+
+The name of the user with sufficient permission to access the database.
+Default value: `zonemaster`.
```

Comment:
I don't think it's right to say that "zonemaster" is the default value of the "user" property. In my view "zonemaster" is just the value we're using in the provided template config file.

Instead I'd say that the "default value" is the value a property takes on if it's omitted from the config file. The presence of a default value for a property indicates that the property is optional.

Technically Z::B::Config::DB_user returns `undef` if "user" is omitted. But this isn't really the default value for the database user, as that causes [DBI::connect](https://metacpan.org/pod/DBI#connect) to take its username value from the `DBI_USER` environment variable. The documentation doesn't specify what default value is used if `DBI_USER` is undefined.

I'm not sure how I think we ought to treat the "user" property with regard to optionality. One possibility is to make it required if "engine" is "MySQL" or "PostgreSQL" and forbidden if "engine" is "SQLite". But I guess that's a topic for another day.

Reply:
I agree that they are not default values in the code, but they are default values when using the default `backend_config.ini`. I think it would be fine to say that they are examples and refer to the default `backend_config.ini` and the installation instructions.

Reply:
Can't we just assume that people interested in the factory settings would look them up in the template config file in the dist directory? Perhaps we could just spell out a command somewhere for how to find the file? I don't think we should go any further than that.

Reply:
It could be enough to refer to the factory settings in the default `backend_config.ini`.

Reply:
With commit https://github.com/zonemaster/zonemaster-backend/pull/684/commits/efeed67bd2105ddf58fcd0433acaa5eb40289d25 I have added a mention at the beginning of the file about the factory settings, and removed the default values.